publish.js

```diff
@@ -15,7 +15,7 @@ const rl = readline.createInterface({
 function run(command, options) {
     const continueOnErrors = options && options.continueOnErrors
     const ret = shell.exec(command, options)
-    if (!continueOnErrors && ret.code !== 0) {
+    if (!continueOnErrors && ret.code) {
```

Comment:
Apparently, not https://github.com/shelljs/shelljs/blob/master/src/exec.js#L65.

Reply:
I think it's safe. Looking at shelljs example, they do the same thing ... https://github.com/shelljs/shelljs#examples

Reply:
It can be safe, but the previous version was more explicit and I believe that the change has worsened rather than improved the code's readability.

Reply:
Worsened? It's kinda good to expect explicit 0 code, otherwise, false, empty string and several others are considered as a success which doesn't make much sense.

Reply:
> It's kinda good to expect explicit 0 code

The explicit 0 code was in the previous version of code,  🤔

Reply:
Ah, sorry, my bad. Ok, going to revert this for now and if @ali-master wants to really earn his badge, send in fixed version.

Reply:
Thanks!

Reply:
@FredyC Yes, of course, I'll revert that to `ret.code !== 0`